src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -293,6 +305,7 @@ public List<DataFile> saveAndAddFilesToDataset(DatasetVersion version,
 					// Any necessary post-processing:
 					// performPostProcessingTasks(dataFile);
 				} else {
+                                    System.out.println("driver is not tmp");
```

Comment:
Do we want to user logger here? warning or something else?

Reply:
Whoops! Fixed (removed) in 8cead71.